.gitattributes

```diff
@@ -4,3 +4,9 @@ sharktank_models/clip/assets/text_model/toy/f32_parameters.irpa filter=lfs diff=
 sharktank_models/clip/assets/text_model/toy/forward_bs4_arg0_input_ids.irpa filter=lfs diff=lfs merge=lfs -text
 sharktank_models/clip/assets/text_model/toy/forward_bs4_expected_result0_last_hidden_state_f32.irpa filter=lfs diff=lfs merge=lfs -text
 sharktank_models/clip/assets/text_model/toy/bf16.mlir filter=lfs diff=lfs merge=lfs -text
+sharktank_models/punet/sharded_toy_resnet_block/assets/expected_results.irpa filter=lfs diff=lfs merge=lfs -text
+sharktank_models/punet/sharded_toy_resnet_block/assets/input_args.irpa filter=lfs diff=lfs merge=lfs -text
```

Comment:
Rather than add individual files and possibly forget which are managed by LFS and which aren't, how about using wildcard patterns?

We've used these in other repositories (https://github.com/nod-ai/SHARK-TestSuite/blob/454722cb4213bc4cd895d152fc755a02b6d8bcf8/iree_tests/.gitattributes):
```
*.irpa filter=lfs diff=lfs merge=lfs -text
*.mlirbc filter=lfs diff=lfs merge=lfs -text
```

Could do something like `**/assets/**/*.irpa` (would need to check the exact syntax)

You could also move this .gitattributes under `sharktank_models/` if we want different rules per folder / test suite